### PR TITLE
[Frontend_v2] Baseline feature

### PR DESCRIPTION
**Pull request by Kajol-Kumari**
**Changes Introduced through this PR**

- [x] Challenge hosts can now make their submissions `baseline` from front end.

- [x] A message telling participants that their submission will still be visible to challenge host will be shown while changing the submission visibility.

GIF:
baseline

GIF after modifying the messages shown for changing submission visibility:
update_dialogue-for-visibility

\cc: @lunayach @Sanji515 @RishabhJain2018 @Shekharrajak

### CODE REVIEW

**Comment by RishabhJain2018** [comment]
@Kajol-Kumari Can you please resolve the conflicts here?

**Comment by lunayach** [comment]
Not exactly related to this PR, but in the demo GIF, it does not make sense to display "host can still see your submission" to the host :sweat_smile:. 
Also, this message should only be displayed when I'm toggling off the visibility option.

**Comment by Kajol-Kumari** [comment]
> Not exactly related to this PR, but in the demo GIF, it does not make sense to display "host can still see your submission" to the host .
> Also, this message should only be displayed when I'm toggling off the visibility option.

Agree! will try to modify the message to be shown while making the submission private and not to the host of the challenge.Thanks @lunayach for pointing this.

File: frontend_v2/src/app/components/challenge/challengesubmissions/challengesubmissions.component.ts [hunk 1]
```diff
@@ -389,20 +389,45 @@ export class ChallengesubmissionsComponent implements OnInit, AfterViewInit {
    */
   changeSubmissionVisibility(id, is_public) {
     is_public = !is_public;
-    this.updateSubmissionVisibility(id);
+    console.log('is Public', is_public);
```
Comment (RishabhJain2018) [inline frontend_v2/src/app/components/challenge/challengesubmissions/challengesubmissions.component.ts:392]: Let's remove this console statement.

Comment (Kajol-Kumari) [reply frontend_v2/src/app/components/challenge/challengesubmissions/challengesubmissions.component.ts]: Done @RishabhJain2018

**Pull request merged**